src/App.tsx

```diff
@@ -1,61 +1,132 @@
+/* eslint-disable max-len */
+import { FormEventHandler, useState, ChangeEvent } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import {
+  getUser, getNewId, getTodos, validateTitleInput,
+} from './utils/appHelper';
```

Comment:
This way it is more readable and easier to maintain
```suggestion
import {
  getUser, 
  getNewId, 
  getTodos, 
  validateTitleInput,
} from './utils/appHelper';
```